diff -u --recursive ../../clean/coreboot-4.13/Makefile.inc 683d5b969e75b346/Makefile.inc
--- ../../clean/coreboot-4.13/Makefile.inc	2020-11-20 12:01:35.000000000 +0000
+++ 683d5b969e75b346/Makefile.inc	2022-04-01 18:28:27.829549464 +0000
@@ -485,7 +485,7 @@
 # Disable style checks for now
 ADAFLAGS_common += -gnatyN
 
-LDFLAGS_common := --gc-sections -nostdlib -nostartfiles -static --emit-relocs
+LDFLAGS_common := --gc-sections -nostdlib -static --emit-relocs
 
 ifeq ($(CONFIG_WARNINGS_ARE_ERRORS),y)
 CFLAGS_common += -Werror
